### DISCUSSION TITLE
RedisJson support

### DISCUSSION THREAD

**Discussion by nsmosi** [redis/go-redis#2187] [General]
Hello, I am wondering when you are going to add RedisJson support please ?

https://redis.io/docs/stack/json/#redis-cli-examples

**Comment by KnutZuidema** [comment]
If this is something that you would consider adding in this library I'd be willing to start working on it, since I am looking for a cluster-aware library with RedisJSON support.
@vmihailenco?

**Reply by vmihailenco** [reply]
Sure, RedisJSON support is welcome.
